packages/web/src/services/audius-sdk/auth.ts

```diff
@@ -53,7 +53,13 @@ export const sdkAuthAdapter = {
     await hedgehogInstance.waitUntilReady()
     return hedgehogInstance.wallet?.getAddressString() ?? ''
   },
-  hashAndSign: async (_data: string) => {
-    return 'Not implemented'
+  hashAndSign: async (data: string) => {
```

Comment:
Is this not also needed on mobile?

Reply:
Oops yes good catch, forgot to test this piece on mobile